x/oracle/abci_test.go

```diff
@@ -42,11 +42,11 @@ func clearHistoricPrices(
 	ctx sdk.Context,
 	k keeper.Keeper,
 	denom string,
-	) {
-	stampPeriod	:= int(k.HistoricStampPeriod(ctx))
+) {
+	stampPeriod := int(k.HistoricStampPeriod(ctx))
```

Comment:
can you check who is the original author of this code and see with him if he has gofmt configured properly?